os-server/src/main/resources/messages.properties

```diff
@@ -0,0 +1,6 @@
+validation.errors=validation errors
+password.error.massage=Password should have 8-20 characters, including upper and lower case letters, numbers and signs(~!@#$%^&*()_{})
+firstName.error.massage=First name is mandatory
+email.error.massage=Email is mandatory
+email-already-exist-exception=User with this email already exists
```

Comment:
please try to be consistent and use one pattern for all messages, 
and please correct the massage -> message
e.g.
duplicate.email.error.message=
email.validation.error.message= 
validation.errors.message=

etc.